### PR TITLE
Fix/company-owner-permission

### PR DESCRIPTION
**Pull request by ulianadzoba**
# [Fix/company-owner-permission](https://issueurl)

Delete Guest class, remove units pagination

- Only signed in user has access to unit actions
- Only company owner can create, edit, delete units
- Signed in user has ability to see unit index and show pages, but doesn't have ability to create, edit, delete (later it will be realized as company member)

## Check list:
- [ ] Integration Tests
- [ ] Functional
- [ ] Unit
- [ ] Factories
- [ ] Updated seed file

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -5,10 +5,6 @@ class ApplicationController < ActionController::Base
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
   rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
 
-  def current_user
```
Comment (Kmalynss) [inline app/controllers/application_controller.rb:8]: Це заьирати поки не треба.

File: app/views/layouts/_nav.html.slim [hunk 2]
```diff
@@ -21,7 +21,7 @@ nav.navbar.navbar-expand-lg.navbar-light.bg-light.shadow.fixed-top
           a.nav-link[href="/contact"]
             | Contact
 
-        - unless current_user.is_a?(Guest)
+        - if current_user
```
Comment (Kmalynss) [inline app/views/layouts/_nav.html.slim:24]: От саме через такі речі потрібен `Guest`.

Comment (ulianadzoba) [reply app/views/layouts/_nav.html.slim:24]: Тому Guest лишаємо?

Comment (Kmalynss) [reply app/views/layouts/_nav.html.slim:24]: Поки так.

**Comment by Kmalynss** [comment]
Закриваю цей ПР.

**Pull request closed without merging**